app/javascript/components/App.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+
+import { Provider } from "react-redux";
+
+import Greeting from './Greeting';
+
+import configureStore from "../redux/configureStore";
+const store = configureStore();
+
+class App extends React.Component {
+  render () {
+    return (
+    <Provider store={store}>
+      <BrowserRouter>
+        <Routes>
+          <Route path='/' element={<Greeting title={'first greeting page path'} />} />
+          {/* <Route path='/greeting' element={<Greeting title={'greeting greeting page path'} />} /> */}
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it make your `code base` dirty and less readable.